spec/requests/resources_spec.rb

```diff
@@ -133,33 +133,6 @@
     end
   end
 
-  describe 'getting a resource with included vendor' do
```

Comment:
Same as above.

Reply:
@keathleydavidj : Please check if there are similar tests for provider. For Example: `getting a package with included provider` and `getting a resource with included provider`. If there are, then these can safely be deleted. Else, please adjust them to look for `provider` instead. Hope this helps.